tools/wheels/cibw_before_build_macos.sh

```diff
@@ -21,17 +21,31 @@ if [[ $PLATFORM == "macosx-x86_64" ]]; then
   #GFORTRAN=$(type -p gfortran-9)
   #sudo ln -s $GFORTRAN /usr/local/bin/gfortran
   # same version of gfortran as the openblas-libs and scipy-wheel builds
-  curl -L https://github.com/MacPython/gfortran-install/raw/master/archives/gfortran-4.9.0-Mavericks.dmg -o gfortran.dmg
-  GFORTRAN_SHA256=$(shasum -a 256 gfortran.dmg)
-  KNOWN_SHA256="d2d5ca5ba8332d63bbe23a07201c4a0a5d7e09ee56f0298a96775f928c3c4b30  gfortran.dmg"
+  curl -L https://github.com/isuruf/gcc/releases/download/gcc-11.3.0-2/gfortran-darwin-x86_64-native.tar.gz -o gfortran.tar.gz
+
+  GFORTRAN_SHA256=$(shasum -a 256 gfortran.tar.gz)
+  KNOWN_SHA256="981367dd0ad4335613e91bbee453d60b6669f5d7e976d18c7bdb7f1966f26ae4  gfortran.tar.gz"
   if [ "$GFORTRAN_SHA256" != "$KNOWN_SHA256" ]; then
       echo sha256 mismatch
       exit 1
   fi
 
-  hdiutil attach -mountpoint /Volumes/gfortran gfortran.dmg
-  sudo installer -pkg /Volumes/gfortran/gfortran.pkg -target /
-  otool -L /usr/local/gfortran/lib/libgfortran.3.dylib
+  sudo mkdir -p /opt/
+  # places gfortran in /opt/gfortran-darwin-x86_64-native. There's then
+  # bin, lib, include, libexec underneath that.
+  sudo tar -xv -C /opt -f gfortran.tar.gz
+
+  # Link these into /usr/local so that there's no need to add rpath or -L
+  for f in libgfortran.dylib libgfortran.5.dylib libgcc_s.1.dylib libgcc_s.1.1.dylib libquadmath.dylib libquadmath.0.dylib; do
+    ln -sf /opt/gfortran-darwin-x86_64-native/lib/$f /usr/local/lib/$f
+  done
+  ln -sf /opt/gfortran-darwin-x86_64-native/bin/gfortran /usr/local/bin/gfortran
+
+  # Set SDKROOT env variable if not set
+  # This step is required whenever the gfortran compilers sourced from
+  # conda-forge (built by isuru fernando) are used
```

Comment:
This was a little confusing, so I propose adding: "outside of a conda-forge environment (so it mirrors what is done in the conda-forge compiler activation scripts)"

Reply:
The compiler tarball that @isuruf makes are collections of individual files extracted from conda-forge builds. They require the SDKROOT variable to be set otherwise they don't work. I'll rework the comment.